app/src/test/java/org/oppia/app/testing/OptionsFragmentTest.kt

```diff
@@ -0,0 +1,72 @@
+package org.oppia.app.testing
+
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.FirebaseApp
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.options.OptionsActivity
+import org.oppia.app.options.STORY_TEXT_SIZE
+import org.oppia.app.options.StoryTextSizeActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+
+private const val KEY_STORY_TEXT_SIZE_PREFERENCE_TITLE = "STORY_TEXT_SIZE_PREFERENCE"
```

Comment:
Remove this from here, instead define this in activitiy's companion object and use it it here using, <AcgtivityName>.<CONSTANT_NAME>

Reply:
Done